### PR TITLE
config-updater: populate CMs on b03

### PR DESCRIPTION
**Pull request by petr-muller**
I noticed this while looking into why RPM repo proxies do not start:

```
Unable to attach or mount volumes: unmounted volumes=[repos], unattached volumes=[mirror-enterprise-basic-auth cache kube-api-access-p5b9j repos key]: timed out waiting for the condition
```

/cc @hongkailiu @openshift/test-platform

### CODE REVIEW

File: core-services/prow/02_config/_plugins.yaml [hunk 1]
```diff
@@ -1793,27 +1793,35 @@ config_updater:
         - bastion-ppc64le-libvirt
         build02:
         - bastion-ppc64le-libvirt
+        build03:
+        - bastion-ppc64le-libvirt
```
Comment (hongkailiu) [inline core-services/prow/02_config/_plugins.yaml:1797]: I would leave the bastion stuff for another PR.
We have lots of other objects than CMs.

Comment (petr-muller) [reply core-services/prow/02_config/_plugins.yaml:1797]: OK

Comment (petr-muller) [reply core-services/prow/02_config/_plugins.yaml:1797]: But having the CMs populated would not hurt /shrug

Comment (petr-muller) [reply core-services/prow/02_config/_plugins.yaml]: Addressed

**Review by hongkailiu — APPROVED** [review]
